src/common/include/sirf/common/GeometricalInfo.h

```diff
@@ -90,12 +90,13 @@ class VoxelisedGeometricalInfo :
 	virtual bool operator==(const GeometricalInfo<num_dimensions, num_dimensions>& gi) const
 	{
 		const VoxelisedGeometricalInfo& vgi = (const VoxelisedGeometricalInfo&)gi;
-		float eps = 1e-2;
+		float eps = 0.01;
```

Comment:
could get rid of this one now. not used

Reply:
@KrisThielemans `eps` is used in lines 96 and 97

GHA builds failed because of some data download troubles - should I re-run right away or should I wait a while?